### PR TITLE
Feat : Notes section add and delete note

### PR DESCRIPTION
**Pull request by t007rushi**
Features:
Add a Note 
Delete a Note

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ function App() {
       <Route
         path="/notes"
         element={
-          <ProtectedRoute ProtectedComp={<NotesPage />}></ProtectedRoute>
+          <ProtectedRoute ProtectedComp={
+          <NotesPage />
+        }></ProtectedRoute>
```
Comment (sushil-mod) [inline src/App.js:20]: i liked how you used private routes

File: src/components/Editor/Editor.jsx [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-import { useState } from "react";
 import "./editor.css";
 
-export default function App() {
-  const [convertedText, setConvertedText] = useState("");
-
+export default function Editor({ handleInput }) {
   return (
     <ReactQuill
       theme="snow"
```
Comment (sushil-mod) [inline src/components/Editor/Editor.jsx:8]: i learned new thing here ReactQuill

**Review by sushil-mod — APPROVED** [review]
good code , there are many things which i learned from this code .

File: src/components/Note/Note.jsx [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+        return (
+          <div className="note" key={item._id}>
+            <div>{item.NoteContent.title}</div>
+            <div
+              className="note-descr"
+              dangerouslySetInnerHTML={{ __html: item.NoteContent.description }}
+            />
+            <div className="flex-row note-options-set">
+              <MdOutlineColorLens onClick={() => {}}/>
+              <RiInboxArchiveLine onClick={() => {}} />
```
Comment (MiheerTamkhane) [inline src/components/Note/Note.jsx:22]: the onClicks are empty is their any reason?

**Review by MiheerTamkhane — APPROVED** [review]
Code is good and readable. Good work man
Good to merge

File: src/components/Note/Note.jsx (hunk 3 above)
Comment (t007rushi) [reply src/components/Note/Note.jsx:22]: will add feat in future

**Pull request merged**